crates/compiler/parse/src/expr.rs

```diff
@@ -3110,12 +3078,43 @@ fn stmts_to_defs<'a>(
             }
             Stmt::Expr(e) => {
                 if i + 1 < stmts.len() {
-                    defs.push_value_def(
-                        ValueDef::Stmt(arena.alloc(Loc::at(sp_stmt.item.region, e))),
-                        sp_stmt.item.region,
-                        sp_stmt.before,
-                        &[],
-                    );
+                    if let Expr::Apply(
+                        Loc {
+                            value: Expr::Dbg, ..
+                        },
+                        args,
+                        _,
+                    ) = e
+                    {
+                        if args.len() != 1 {
+                            // TODO: this should be done in can, not parsing!
+                            return Err(EExpr::Dbg(
```

Comment:
Presumably this TODO will be resolved in the next PR?

Reply:
Not the very next PR, but... soon. Before I attempt the Defs->Stmts refactor again, I want to:
* Get fuzzing "clean" on main
* Extend current fuzzing to cover `can`